complex/cargo/list/src/lib.rs

```diff
@@ -205,9 +206,9 @@ impl ListManager {
         conn.execute(item_sql, &[&item_uuid, &item.name, &item.due_date, &item.completion_date]).unwrap();
 
         let item_label_sql = r#"INSERT INTO item_labels (item_uuid, label_name) VALUES (?, ?)"#;
```

Comment:
If you were doing this as a good SQLite citizen, you would insert the item and its labels inside a transaction, and you would have a prepared statement for adding the labels.